### ISSUE TITLE
Tensorboard not writeing all datapoints 

### ISSUE THREAD

**Issue by abdjava** [pytorch/pytorch#36920]
## 🐛 Bug

While training it seems tensorboard allways misses the last datapoint . I have a read of pytorch docmentation https://pytorch.org/docs/stable/tensorboard.html . It seems example code form there does not give as many datapoints as expected . Running it 10 times gives me 0 datapoints and running it 100 times only gives me 88 datapoints .

## To Reproduce

1. writeing random data into a events file useing tensorboard from pytorch 
2. opening it with tensorboard  . I seem to get no scalar data

Useing 

1. pytorch -1.4.0
2. python 3.7.5
3. tensorboard 2.1.0

code 
```python
import numpy as np 
from torch.utils.tensorboard import SummaryWriter

writer = SummaryWriter( )


for i in range(10):
    writer.add_scalar('loss/test',np.random.random() , i)

```
## Expected behavior

should produce a file with exertly 10 data points under loss/test with y values in range [ 0, 1.0 ) and 
 interger x values of  range [ 0,10) .
This should very similar to doc example 

## Environment
 - PyTorch Version (e.g., 1.0): 1.4.0
 - OS (e.g., Linux): redhat 7.6
 - How you installed PyTorch (`conda`, `pip`, source): conda 
 - Python version: 3.7.5
 - CUDA/cuDNN version: 10.1/7.6.0
 - GPU models and configuration: 2080ti 
 - tensorboard - 2.1.0

**Comment by vadimkantorov** [comment]
Related: https://github.com/pytorch/pytorch/issues/24234 (workaround). Unfortunately, it's not fixed yet :(

**Comment by albanD** [comment]
Closing in favor of #24234
